### PR TITLE
Feat: Add GUI and Overall Project Structure for Python Project

### PR DESCRIPTION
**Pull request by MANGRYANG**
### Overview

This PR adds the overall structure of the Python project and implements the GUI for the project.

### Changes Made

- Implemented the overall structure of the project.
- Developed a GUI for user interaction.
- Made necessary code modifications for added functionalities.
- Applied several UI design changes for better user experience.

### Testing Done

The changes have been validated through the following tests:

- [x] User interface testing
- [x] Functional test cases
- [x] Edge case testing

### CODE REVIEW

**Review by Ham3798 — APPROVED** [review]
LGTM

File: bin/vid_to_pdf.py [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+        self.savetoentry = ttk.Entry(url_panel, width=55)
+        set_button = ttk.Button(url_panel, text="Set", command=self.set)
+        folder_button = ttk.Button(url_panel, text="Folder", command=self.folder)
+
+        url_label.grid(row=0, column=0, sticky=tk.W)
+        self.urlentry.grid(row=0, column=1, columnspan=2, pady=10, sticky=tk.W)
+        set_button.grid(row=0, column=3, sticky=tk.W, padx=(5, 0))
+        saveto_label.grid(row=1, column=0, sticky=tk.W)
+        self.savetoentry.grid(row=1, column=1, sticky=tk.W)
+        folder_button.grid(row=1, column=3, sticky=tk.W, padx=(5, 0))
```
Comment (Alencion) [inline bin/vid_to_pdf.py:76]: saveto_label, savetoentry 케이스가 불규칙한 것 같아요. 

컨벤션이 따로 정할걸까요?

**Review by Alencion — COMMENTED** [review]
👍

**Comment by Alencion** [comment]
pr 메시지를 잘 적어두었습니다! 👍 

메시지가 잘 구조화 되어있는데요. 

github pr template을 만들어서 사용했으면 더 좋았을 것 같아요!

[참고](https://2jinishappy.tistory.com/337)
